app/components/post/index.js

```diff
@@ -5,24 +5,28 @@ import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 
 import {createPost, deletePost, removePost} from 'mattermost-redux/actions/posts';
+import {General} from 'mattermost-redux/constants';
+import {getCurrentChannel} from 'mattermost-redux/selectors/entities/channels';
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
 import {getCurrentUserId, getCurrentUserRoles} from 'mattermost-redux/selectors/entities/users';
 import {getMyPreferences, getTheme} from 'mattermost-redux/selectors/entities/preferences';
 import {getCurrentTeamUrl} from 'mattermost-redux/selectors/entities/teams';
 import {isPostFlagged} from 'mattermost-redux/utils/post_utils';
+import {isAdmin, isChannelAdmin, isSystemAdmin} from 'mattermost-redux/utils/user_utils';
 
 import {insertToDraft, setPostTooltipVisible} from 'app/actions/views/channel';
 import {addReaction} from 'app/actions/views/emoji';
 import {getDimensions} from 'app/selectors/device';
 
 import Post from './post';
 
-function mapStateToProps(state, ownProps) {
+function mapStateToProps(state, ownProps) {//eslint-disable-line complexity
```

Comment:
Can you remove these disabling comments? The warnings are annoying, but they're there for a reason

Reply:
Done.